model/src/w3gridmd.F90

```diff
@@ -3613,13 +3613,24 @@ SUBROUTINE W3GRID()
 !
       DO_CHANGE_WLV=.FALSE.
       IF ( GTYPE.EQ.UNGTYPE) THEN 
-        UNSTSCHEMES(:)=0
-        IF (EXPFSN)   UNSTSCHEMES(1)=1
-        IF (EXPFSPSI) UNSTSCHEMES(2)=1
-        IF (EXPFSFCT) UNSTSCHEMES(3)=1
-        IF (IMPFSN)   UNSTSCHEMES(4)=1
+        UNSTSCHEMES = 0
+        IF (EXPFSN)   UNSTSCHEMES(1) = 1
+        IF (EXPFSPSI) UNSTSCHEMES(2) = 1
+        IF (EXPFSFCT) UNSTSCHEMES(3) = 1
+        IF (IMPFSN)   UNSTSCHEMES(4) = 1
+        IF (IMPTOTAL) UNSTSCHEMES(5) = 1
+        IF (EXPTOTAL) UNSTSCHEMES(6) = 1
```

Comment:
UNSTSCHEMES is only allocated to 4 you just reverted the change that increases the bounds to 6.

Reply:
Yes, this was the reason that I have taken all this logic into this commit. We will take it out and commit in a separate PR.